examples/visualization/brain.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+# Plot a brain image
+# ------------------
+#
+# First we plot a brain. This requires a directory with MRI? information.
+# MNE provides functionality to automatically retrieve standard head images.
+# In this example we use the ``sample`` data which is data from an example
+# subject being shown auditory and visual stimuli.
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, background='white', cortex='low_contrast')
+
+
+# %%
+# Add source information
+# ----------------------
+#
+# Plot source information.
+sample_dir = op.join(data_path, 'MEG', 'sample')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+stc.crop(0.09, 0.1)
+
+fmin, fmax = stc.data.min(), stc.data.max()
+for i, hemi in enumerate(('lh', 'rh')):
+    hemi_data = stc.data[:len(stc.vertices[i])]
+    hemi_vertices = stc.vertices[i]
+    brain.add_data(hemi_data, fmin=fmin, hemi=hemi, fmax=fmax,
+                   vertices=hemi_vertices, alpha=0.5,
+                   smoothing_steps='nearest', time=stc.times)
+
+
+# %%
+# Modify the view of the brain
+# ----------------------------
+#
+# You can adjust the view of the brain using ``show_view`` method.
+brain.show_view(azimuth=190, elevation=70, distance=500)
+
+
+# %%
+# Highlight a region on the brain
+# -------------------------------
+#
+# It can be useful to highlight a region of the brain for analyses.
+# To highlight a region on the brain you can use the ``add_label`` method.
+# Labels are stored in the Freesurfer label directory for that subject.
```

Comment:
This is still unclear to me as a new user. Do I need to then install freesurfer to have that directory? Maybe adding the default location or link to another page discussing this?

Reply:
They come with the recon, we could mention that. You can also make labels however you wish, which is described in the freesurfer documentation https://surfer.nmr.mgh.harvard.edu/fswiki/mri_vol2label